cache/releases/keycloak-client/26.0.1/gh-release-notes.html

```diff
@@ -0,0 +1,20 @@
+<div>
+
+<h2>Upgrading</h2>
+<p>Before upgrading refer to <a href="https://www.keycloak.org/docs/latest/upgrading/index.html#migration-changes">the migration guide</a> for a complete list of changes.</p>
```

Comment:
This link should probably go to https://www.keycloak.org/securing-apps/upgrading ? 

I am pretty sure that location https://www.keycloak.org/docs/latest/upgrading/index.html#migration-changes is not correct as it contains server-specific upgrading notes related to the Keycloak server. Right now, we don't have any version specific upgrades for keycloak-client in https://www.keycloak.org/securing-apps/upgrading , but we can possibly add them to this file in the future if we have some backwards incompatible changes needed for the client. That file is rendered from the keycloak-client repository ( https://github.com/keycloak/keycloak-client/blob/main/docs/guides/securing-apps/upgrading.adoc ), so we can easily add any client specific updates there.

Reply:
I've added some logic to allow setting the path to the upgrade guide per source.